### PR TITLE
docs(connector): add fb readme zh-CN

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

Add FB README zh-CN

## Testing

N/A

### CODE REVIEW

File: packages/connector-facebook/README.md [hunk 1]
```diff
@@ -1,63 +1,155 @@
...
+1. 在 Facebook 应用「控制面板」，从侧边栏，点按「设置」->「基本」。
+2. 在面板中你会看到 _应用编号_ 和 _应用密钥_。
+3. 点按在「应用秘钥」右边的「显示」按钮，然后复制内容。
+4. 填写 Logto 连接器设置：
+   - 用 _应用编号_ 里找到的信息填写 `clientId` 栏。
+   - 用 _应用密钥_ 中的信息填写 `clientSecret` 栏。
+
+## 用 Facebook 的测试用户来测试登录
+
+你可以用测试帐号、开发者帐或者管理员帐号在开发环境或者真实环境下测试相关应用的登录流程。了解 [应用模式](https://developers.facebook.com/docs/development/build-and-test/app-modes)。
```
Comment (demonzoo) [inline packages/connector-facebook/README.md:125]:
```suggestion
你可以用测试帐号、开发者帐号或者管理员帐号在开发环境或者真实环境下测试相关应用的登录流程。了解 [应用模式](https://developers.facebook.com/docs/development/build-and-test/app-modes)。
```

File: packages/connector-facebook/README.md [hunk 2]
```diff
@@ -1,63 +1,155 @@
...
+
+| 名称         | 类型   |
+|--------------|--------|
+| clientId     | string |
+| clientSecret | string |
+## 参考
+
+- [Facebook 登录 - 文档 - 面向开发者](https://developers.facebook.com/docs/facebook-login/)
+    - [手动构建登录流程](https://developers.facebook.com/docs/facebook-login/guides/advanced/manual-flow/)
+    - [Facebook 登录的相关权限](https://developers.facebook.com/docs/facebook-login/guides/permissions)
```
Comment (demonzoo) [inline packages/connector-facebook/README.md:155]: Append `?locale=zh_CN` to URLs

**Review by demonzoo — APPROVED** [review]
Overall looks good

File: packages/connector-facebook/README.md (hunk 2 above)
Comment (darcyYe) [reply packages/connector-facebook/README.md:155]: This is per testing, to ensure that Chinese readme goes to Chinese document website.

**Pull request merged**